docs/intro.md

```diff
@@ -0,0 +1,120 @@
+# Introduction to Scipian
+
+## What is Scipian?
+
+Scipian is an open source platform to give teams a single, self-service place to
+access all the tools and services needed to develop and deploy applications. To
+put it more simply, it's a continuous delivery platform that combines disparate
+tools and utilities to create affinity around developing and deploying
+applications. It is not tools/platforms like GitHub Enterprise, Artifactory,
+Kubernetes, or OpenStack.
+
+The value stream behind Scipian is to focus on speed and agility - bringing new
+innovation and technology to support business strategy. Scipian is propelling
+the engineer's to build applications by enabling engineers to focus on what
+they do best - write code, operations, or administration.
+
+Definition:
+> The word `Scipian` originates from Old English, meaning to send or transport
+(merchandise, people) by ship and is the root of [ship][ship-root].
+
+[ship-root]: http://www.etymonline.com/index.php?allowed_in_frame=0&search=scipian
+
+## Why Scipian?
+
+### History
+
+As with anything, when working with legacy systems and platforms inside an
+enterprise business, we needed to understand the past before we can solve for
+the future.
+
+#### In the beginning... I think I need a server?
+
+  1. Engineer team needs to build a new application
+  1. Reach out to infrastructure team
+  1. Infrastructure team places an order for servers
+  1. Wait some time ...
+  1. Servers arrive
+  1. Infrastructure team racks the servers
+  1. Infrastructure turns the servers over to the engineering team
+
+Sounds simple, but at the same time very complex. We've abstracted some of this
+with virtualization and handed API's over to the teams to help mitigate the
+steps above. However, it was only incremental benefits and it introduced other
+problems.
+
+#### Infrastructure Change Fatigue
+
+Engineering teams who got their pet servers spent a lot of time patching, even
+with new things like OpenStack and etc., we still asked teams to do a lot of
+work to make changes to support the platform, upgrades, migrations, and etc.
+This infrastructure work culminated into the teams backlogs and it's been
+destructive for them.
+
+The engineering teams were not alone, as the infrastructure teams spent a lot of
+time tracking abandoned and new servers, doing server upgrades, and etc.
+
+The true loss here was the customer, as teams weren't agile enough to get
+something to market fast enough.
+
+#### Disparate Experiences
+
+From an product engineering standpoint, teams have a very disparate experience.
+They need to deploy applications to internal data centers and a cloud provider
+like Amazon Web Services, Google Cloud Platform, and Microsfot Azure. Deployment
+is very differnt between the options, which caused the engineering teams to
+create deployment tools, which meant they had to build them and maintain the
+toolchains. This can reach up to 70% of their time to get a deployment out for
```

Comment:
question: Not arguing with this 70% figure, but is there any data we could link to for it?

Reply:
Currently it's an arbitrary number, I removed it for now and added a TODO to replace once researched more with links to back it up. That was the plan anyway, might as well track it that way. 😄